### PR TITLE
Post02

### PR DESCRIPTION
**Pull request by Alina-PANG**
1. edit the message detail page css
2. backend function fixes
3. backend + front end integration

currently it is able to achieve: 
1. comment: create, view all
2. like: create, delete
3. post: view detail

### CODE REVIEW

**Comment by Alina-PANG** [comment]
Screen Shot 2019-07-06 at 9 29 51 AM

File: src/main/webapp/feed.html [hunk 1]
```diff
@@ -34,6 +34,16 @@ <h2> Play Even Harder </h2>
     <div id="content" class="container">
       <h1>Message Feed</h1>
       <hr />
+      <div id="feed-create-container">
```
Comment (nanda-thant-sin) [inline src/main/webapp/feed.html]: What is the use case of this?

Comment (Alina-PANG) [reply src/main/webapp/feed.html]: for the UI, I added a creation form for creating new post at the post feed page

Comment (nanda-thant-sin) [reply src/main/webapp/feed.html]: So, is it supposed to works like form from `User-Page.html`?

**Pull request merged**